dashboard/test/controllers/home_controller_test.rb

```diff
@@ -362,7 +362,7 @@ class HomeControllerTest < ActionController::TestCase
   # TODO: remove this test when workshop_organizer is deprecated
   test 'workshop organizers see dashboard links' do
     sign_in create(:workshop_organizer, :with_terms_of_service)
-    query_count = 14
+    query_count = 15
```

Comment:
I don't really understand why we're asserting the query count here but <shrug>.

Reply:
Yeah... I'm not sure either